rusk/src/lib/services/network.rs

```diff
@@ -187,13 +193,16 @@ mod serialization {
     const MAGIC_BYTES: [u8; 4] = MAGIC_DEVNET.to_le_bytes();
     const TX_CATEGORY: u8 = 10;
 
-    const RESERVED_FIELDS_BYTES: [u8; 8] = [0; 8];
-    const TX_HEADER_LEN: u64 = {
-        4u64 + // MAGIC
-        8u64 + // RESERVED_FIELDS
-        4u64 // CHECKSUM
+    const RESERVER_FIELDS_LEN: usize = 8;
+    const RESERVED_FIELDS_BYTES: [u8; RESERVER_FIELDS_LEN] =
+        [0; RESERVER_FIELDS_LEN];
+    const TX_HEADER_LEN: usize = {
+        u32::SIZE + // MAGIC
+        RESERVER_FIELDS_LEN + // RESERVED_FIELDS
+        32 // CHECKSUM
```

Comment:
32 bytes checksum sound quite exaggerated. You sure of this? Or is this a weirdly named hash?

Reply:
It was wrong, the checksum is just the first 4 bytes of the hash